packages/eth-withdraw/README.md

```diff
@@ -1,44 +1,38 @@
-# eth-withdraw Tutorial
+# eth-withdraw tutorial
 
-`eth-withdraw` shows how to move Ether from Arbitrum (Layer 2) into the Ethereum (Layer 1) chain.
+`eth-withdraw` shows how to move Ether from an Arbitrum or Orbit chain into its parent chain.
```

Comment:
This can also be used to withdraw native token, how about we add a sentence here to say sth like`if your orbit chain uses native token, you can also use this way to withdraw native token`?

Reply:
Good catch! I'll add that

Reply:
Modified all mentions to Ether, to also include the native asset of an Orbit chain